python_examples/sinric/_cbhandler.py

```diff
@@ -1,32 +1,80 @@
+import logging
+
 
 class CallBackHandler:
     def __init__(self, callbacks):
         self.callbacks = callbacks
 
     def handle_callbacks(self, jsn):
-        if jsn['action'] == 'setPowerState':
+        if jsn.get('action') == 'setPowerState':
```

Comment:
The use of `get` function to avoid `KeyError` exceptions.
![image](https://user-images.githubusercontent.com/7910856/61361118-244fca80-a880-11e9-927b-b6e40b75de07.png)